R/zzz.R

```diff
@@ -1,9 +1,9 @@
 .onLoad <- function(libname, pkgname) {
   op <- options()
   op.bfast <- list(
-    bfast.prefer_matrix_methods = FALSE,
-    bfast.use_bfastts_modifications = FALSE,
-    strucchange.use_armadillo=FALSE
+    bfast.prefer_matrix_methods = TRUE,
+    bfast.use_bfastts_modifications = TRUE,
```

Comment:
This depends on #19 though. And good to include a more complex test in the bfastpp examples, perhaps, that would cover regular and irregular cases.